implementations/rust/ockam/ockam_api/src/logs/setup.rs

```diff
@@ -90,7 +90,7 @@ impl LoggingTracing {
             )
         };
         info!(
-            "tracing initialized. {} {}",
+            "tracing initialized. {:#?} {:#?}",
             logging_configuration, tracing_configuration
```

Comment:
Perhaps we could split this in two to avoid polluting too much the `info` channel:

```
info!("tracing initialized");
debug!(%logging_configuration, %tracing_configuration);
```

Reply:
done